src/test/data/key_io_invalid.json

```diff
@@ -156,25 +156,25 @@
         "bc1qw508d6qejxtdg4y5r3zarvary0c5xw7kv8f3t5"
     ],
     [
-        "BC13W508D6QEJXTDG4Y5R3ZARVARY0C5XW7KN40WF2"
+        "DG13W508D6QEJXTDG4Y5R3ZARVARY0C5XW7K4P8H7H"
     ],
     [
-        "bc1rw5uspcuh"
+        "dcrt1rw53fe0sl"
     ],
     [
-        "bc10w508d6qejxtdg4y5r3zarvary0c5xw7kw508d6qejxtdg4y5r3zarvary0c5xw7kw5rljs90"
+        "dcrt10w508d6qejxtdg4y5r3zarvary0c5xw7kw508d6qejxtdg4y5r3zarvary0c5xw7kw5056q2h"
     ],
     [
-        "BC1QR508D6QEJXTDG4Y5R3ZARVARYV98GJ9P"
+        "DG1QR508D6QEJXTDG4Y5R3ZARVARYVGU055V"
     ],
     [
         "tb1qrp33g0q5c5txsp9arysrx4k6zdkfs4nce4xj0gdcccefvpysxf3q0sL5k7"
     ],
     [
-        "bc1zw508d6qejxtdg4y5r3zarvaryvqyzf3du"
+        "dg1zw508d6qejxtdg4y5r3zarvaryvqndalvf"
```

Comment:
This one looks wrong to me. it will fail for the wrong reason?

Reply:
Good catch, fixed